### PR TITLE
LOG-5884: allow empty URL in validation rules for CloudWatch and Kafka

### PR DESCRIPTION
**Pull request by vparfonov**
### Description

/cc @Clee2691 @cahartma @alanconway 
/assign @jcantrill 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA: https://issues.redhat.com/browse/LOG-5884
- Enhancement proposal:

### CODE REVIEW

File: test/e2e/collection/apivalidations/api_validations_test.go [hunk 1]
```diff
@@ -85,5 +82,14 @@ var _ = Describe("", func() {
 		Entry("should fail with invalid name", "invalid_name.yaml", func(out string, err error) {
 			Expect(err.Error()).To(MatchRegexp("Name.*valid DNS1035"))
 		}),
+		Entry("should pass for Cloudwatch with no URL", "cloudwatch-no-url.yaml", func(out string, err error) {
```
Comment (jcantrill) [inline test/e2e/collection/apivalidations/api_validations_test.go:85]: where are these files?

Comment (vparfonov) [reply test/e2e/collection/apivalidations/api_validations_test.go:85]: added

**Pull request merged**